dags/surveymonkey_data_import_pipeline.py

```diff
@@ -117,6 +119,39 @@ def surveymonkey_survey_questions_etl(**kwargs):
         )
 
 
+def surveymonkey_survey_answers_etl(**kwargs):
+    data_config = data_config_from_xcom(kwargs)
+    survey_id_list_from_df = get_distinct_values_from_bq(
+        project_name=data_config.project_name,
+        dataset_name=data_config.dataset_name,
+        table_name_source=data_config.survey_list_table_name,
+        column_name=data_config.survey_id_column_name
+    ).values.tolist()
+
+    for survey_id_list in survey_id_list_from_df:
+        survey_id = str(survey_id_list[1])
+        LOGGER.info("answers for survey_id: %s", survey_id)
+        list_of_answers_of_one_survey = get_survey_answer_details(
+            access_token=get_surveymonkey_access_token(),
+            survey_id=survey_id
+        )
+        len_of_answer_list = len(list_of_answers_of_one_survey)
+        LOGGER.info("length of answers is %s", len_of_answer_list)
+        if len_of_answer_list > 0:
+            for answers_dict in list_of_answers_of_one_survey:
+                survey_answers_list = [
+                    get_bq_json_for_survey_answers_response_json(answers_dict)
+                ]
+                load_given_json_list_data_from_tempdir_to_bq(
+                    project_name=data_config.project_name,
+                    dataset_name=data_config.dataset_name,
+                    table_name=data_config.survey_answers_table_name,
+                    json_list=survey_answers_list
+                )
+        else:
+            LOGGER.info("There is no answer for the survey %s", survey_id)
```

Comment:
minor: to reduce nesting, one common practise is to use early stopping, e.g.:

```python
if not list_of_answers_of_one_survey:
    LOGGER.info("There is no answer for the survey %s", survey_id)
    continue
for answers_dict in list_of_answers_of_one_survey:
    # ...
```

Although in this case the `continue` may not even be necessary if we are just "looping" over an empty list.